okhttp/src/jvmMain/kotlin/okhttp3/internal/cache/FaultHidingSink.kt

```diff
@@ -53,9 +53,9 @@ internal open class FaultHidingSink(
   }
 
   override fun close() {
-    if (hasErrors) {
-      return
-    }
+//    if (hasErrors) {
```

Comment:
This stops the actual close from working.

We either close anyway here, or close on error.

The latter probably means a change to FaultyFileSystem?